### DISCUSSION TITLE
Apenas uma DEMO da minha aplicаção.

### DISCUSSION THREAD

**Discussion by alexptbg** [LITUATUI/M3UPT#76] [General]
Eu comecei este projecto á 2 anos atrás, pela mesma razão, ajudar a levar a TV portuguesa á casa de portugueses que estão no estrangeiro.
Não foi por questóes de pagar ou não pagar, ser legal ou ilegal; foi por questões de burocracia e/ou dificuldade de aceder a certos serviços ou pagar certos serviços fora de Portugal, ou de taxas além da conta.
  
Eu vivo na Bulgaria á + de 10 anos, por isso na APP está em destaque os países PT e BG.

Eu sou programador de hardware, firmware & software á + de 20 anos. 
Saí de Portugal em 2007. Andei a trabalhar na Holanda, Bélgica, Alemanha e agora Bulgaria.

É baseado em IPTV STB (stalker portal iptv).
No momento estou a usar 4 servidores fisicos para a generação dos links iptv e + 1 servidor fisico para scanear os endereços MAC por portal automaticamente.
A versão original contem mais de 160 portais que mudam semanalmente, com acesso a mais de 20.000 canais de todo o mundo + filmes & séries com legendas, inclusive e TV networks (netflix, amazon, apple tv, disney, etc).

Se quiserem dar uma olhada:

https://eesystems.net/demo/

Codigo pin : 00000

Todos os links gerados deverão ser abertos "automaticamente" no VLC (ou outro video player) porque o link gerado contem uma "hash" que e válida apenas por alguns segundos. 

Se algum portal não funcionar, tentem outro.
Se algum link não funcionar, tentem novamente. (tip: activem o repeat[1] no video player).

Funciona em qualquer sistema operativo (android, windows, linux, mac, iOS).

Estou a fazer este post não é por questões de ADS/spam, é apenas para idea ou dica, ou que estou disposto ajudar com alguma coisa caso precisem.

O acesso a minha APP é grátis. Apenas cobro o dinheiro na venda da TV box. que é +- 60 euros.

A minha TV box e baseada no RASPBERRY PI + DEBIAN (minimal install) + CHROME em KIOSK MODE (sem menus e barras = como se fosse uma webview) + VLC, sem desktop, e carrega directamente a app url do servidor principal. Assim facilita o update das TV boxes quando e uma app remota.

A APP como é especialmente dedicada ao RASPBERRY PI não existe login, tem mais algumas opções, como restart no raspberry, virtual keyboard, etc... 

Pretendo daqui a 1 ou 2 anos desenvolver a segunda versão da APP, com um visual mais apelativo e estou a espera dos browser's suportarem video native.

Agradeço o feedback.

**Comment by thomraider12** [comment]
Olá, é muito bom, mas tenho um amigo na Suíça que por motivos de geolocalização não consegue nem abrir a RTP, etc. 
Mas seria uma grande ideia.
